docs/contributing/999-rfcs/2023-06-12-language-spec.md

```diff
@@ -776,15 +777,121 @@ let handler2 = inflight() => {
 }
 ```
 
-Bridge between preflight and inflight is crossed with the help of immutable data
-structures, "structs" (user definable and `Struct`), and the capture mechanism.
+The bridge between preflight and inflight is crossed with the help of immutable data
+structures, user-defined structs, resources, and the capture mechanism.
 
 Preflight class methods and initializers can receive an inflight function as an argument. This
 enables preflight classes to define code that will be executed on a cloud compute platform such as
 lambda functions, docker, virtual machines etc.
 
 [`▲ top`][top]
 
+#### 1.3.1 Phase-independent code
+
+> **Note**: Phase-independent functions are not yet implemented. Subscribe to [issue #435](https://github.com/winglang/wing/issues/435) for updates.
+
+Code that is not dependent on the phase of execution can be designated as phase-independent using the `unphased` modifier.
+
+Using this modifier means that the function can be called from either preflight or inflight contexts.
+
+```TS
+let odd_numbers = unphased (arr: Array<num>): Array<num> => {
```

Comment:
I like the suggestion but I prefer `inflight?` instead of introducing another keyword to the language.

I think it reads nicely:

```js
let foo = inflight? () => {
  return 34;
}
```

Reply:
With just `inflight`, `preflight`, `unphased` perspective I think it reads nice, but my concern here is the first thing I think of when I see `inflight?` is an optional inflight.

(Thinking outloud)
The RFC mentions:
```
However, a preflight or inflight function cannot be passed to a function that expects an unphased function.
```

How would an expected unphased function look in a method signature if we use `inflight?`

Reply:
I'm a little worried there could be a confusion or mental overloading with "?" since "foo?" looks a lot like an optional type. (It's also handy that "unphased" is fewer characters). But I'm not sure - let's keep the option open, it's probably the easiest part of the design to change 👍

Reply:
I think that “optional inflight” is actually conveying the idea of “unphased” pretty well.

It implies that this is a preflight function that can also be used inflight.

We should be very careful to add additional keywords to the language. Every new keyword is a huge cognitive overload.

Reply:
I agree that thinking of the function as "optionally inflight" means it could be inflight or preflight.

However, we define optionality in the language as  potentially having a "lack of value" so T? is [T || nil], which makes `inflight?` being [inflight || preflight] weird in my opinion.